contents/gpgpu/gpgpu.md

```diff
@@ -2,6 +2,8 @@
 
 ###### Author: [Pierce Anderson Fu](https://github.com/PierceAndy)
 
+###### Author: [Nguyen Quoc Bao](https://github.com/bqnguyen94)
```

Comment:
You could combine the two lines of authors. See: https://github.com/se-edu/learningresources/pull/37/files#diff-e237ae1cc0dfe433d7b40fab5d5775c8R3